modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/BasicIndexTest.java

```diff
@@ -70,6 +98,17 @@
 
         igniteCfg.setConsistentId(igniteInstanceName);
 
+        if (igniteInstanceName.startsWith(CLIENT_NAME)) {
+            igniteCfg.setClientMode(true);
+
+            if (clntLog != null)
```

Comment:
What does `clnt` mean? There is no such abbreviation in abbreviation rules (https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules).